contracts/truefi/LoanToken.sol

```diff
@@ -158,7 +163,7 @@ contract LoanToken is ILoanToken, ERC20 {
     /**
      * @dev Only when loan is Settled
      */
-    modifier onlyClosed() {
+    modifier onlySettled() {
```

Comment:
I think modifier's name should remain `onlyClosed()`
`onlySettled()` is not accurate, because `status >= Status.Settled` means settled/defaulted/liquidated. 
In all of those cases redeem should be an option, in case when borrower repaid his debt partially
Also forbidding reclaim when a loan defaulted, would just freeze funds forever and nobody would profit from that

Reply:
My bad, thanks for catching my mistake. I'll revert this change then. I have to read more carefully!

Reply:
I've reverted the name to onlyClosed() and updated the documentation to mention Defaulted + Liquidated as well.